src/stream_writer.rs

```diff
@@ -0,0 +1,54 @@
+#![allow(dead_code)]
+use tokio::net::tcp::OwnedWriteHalf;
+use tokio::sync::mpsc::Receiver;
+use tokio::task::JoinHandle;
+
+use crate::message_types::{EarlyReply, EarlyResult, ProcResult, Reply};
+
+/// Writes [`vfs_task::VfsTask`] responses to a network connection.
+pub struct StreamWriter {
```

Comment:
Why stream writer? Why ReadTask still ReadTask, not StreamRead task? For what `Stream` stands for?